lib/features/registration/pages/signup_page.dart

```diff
@@ -367,12 +372,13 @@ class _SignUpPageState extends State<SignUpPage> {
     );
   }
 
-  Widget _createView(StripeObjectStatus status) {
+  Widget _createView(StripeObjectStatus status, UserExt user) {
     if (status == StripeObjectStatus.loading) {
       return const Center(child: CircularProgressIndicator());
     }
     if (status == StripeObjectStatus.failure) {
-      return const Text('Could not connect to Stripe');
+      return Text(
+          'Could not connect to Stripe \nuser email: ${user.email} \nuser country: ${user.country}');
```

Comment:
Maybe its better to write info like that to logit. We don't want to show users this email/country thing i guess?